pkg/storage/etcd/etcd_helper.go

```diff
@@ -56,27 +55,32 @@ func (c *EtcdConfig) GetType() string {
 
 // implements storage.Config
 func (c *EtcdConfig) NewStorage() (storage.Interface, error) {
-	etcdClient := etcd.NewClient(c.ServerList)
-	if etcdClient == nil {
-		return nil, errors.New("Failed to create new etcd client from serverlist")
-	}
-	transport := &http.Transport{
-		Dial: forked.Dial,
-		TLSClientConfig: &tls.Config{
-			InsecureSkipVerify: true,
+	cfg := etcd.Config{
+		Endpoints: c.ServerList,
+		// TODO: Determine if transport needs optimization
+		Transport: &http.Transport{
```

Comment:
The defaults are explicitly specified so others will know the values on inspection, vs. hunt and peck.